app/controllers/observations_controller/form_helpers.rb

```diff
@@ -16,20 +16,38 @@
 module ObservationsController::FormHelpers
   private
 
+  # Note this array syntax: The nested attributes MUST come last.
+  # VSCode won't allow nested square brackets (?).
   def permitted_observation_args
     [:place_name, :where, :lat, :long, :alt, :when, "when(1i)", "when(2i)",
-     "when(3i)", :notes, :specimen, :thumb_image_id, :is_collection_location,
-     :gps_hidden]
+     "when(3i)", :specimen, :thumb_image_id, :is_collection_location,
+     :gps_hidden, { notes: [:Other] }]
```

Comment:
So, what I'm still confused about -- sorry! -- is how this ever passed tests.  The log message claimed "notes" was unpermitted.  Surely we have a test which involves creating an observation with notes, no?

Also, why single out :Other here, and none of the other notes fields?  I mean, sure, that's user-defined, so how do you apriori know which notes fields to permit?

Yea, I guess I'm just confused.  Like you say, this must already work... somehow...  It just bugs me when I can't imagine how.  Or at least understand why it's displaying an error message in the log that apparently doesn't mean anything(!)

Again, feel free to ignore and move on to greener pastures.  I have no problem with "don't fix it if it ain't broke".  Just hoping you have some amazing new insight that will make all sorts of heavenly lights go on! :)

Reply:
I'm equally confused. (And just not focussing well this a.m.)